db/migrate/20150625220141_fix_serialized_reports_for_rails_four.rb

```diff
@@ -19,8 +22,10 @@ def serialize_report_to_hash(val)
       YAML.dump(new_hash)
     end
 
-    def serialize_hash_to_report(val, from)
-      if val.starts_with?("---")
+    def serialize_hash_to_report(val, from, migration)
+      if val.starts_with?('--- !')
+        migration.say "#{self.class} Id: #{id} is not a Hash, skipping conversion"
```

Comment:
we might want to remove this if it becomes verbose.  We really should only log exceptions, and avoid anything unnecessarily extra.

For now, it doesn't prevent merge though.